ws-nextjs-app/pages/[service]/send/[id]/SubmitButton/styles.ts

```diff
@@ -0,0 +1,27 @@
+import { Theme, css } from '@emotion/react';
+
+export default {
+  submit: ({ palette, fontSizes, fontVariants, spacings, mq }: Theme) =>
+    css({
+      backgroundColor: palette.SERVICE_NEUTRAL_CORE,
+      color: palette.WHITE,
+      width: '100%',
+      ...fontVariants.sansBold,
+      ...fontSizes.bodyCopy,
+      border: 'none',
+      padding: `${spacings.FULL + spacings.HALF}rem 0`,
+      marginBottom: `${spacings.DOUBLE}rem`,
+      cursor: 'pointer',
+      '&:hover,&:focus-visible': {
```

Comment:
I think this declaration can just be '&:hover, &:focus'. The focus-visible psuedo-class really only needs  to be specified if we're trying to override the browser default behaviour.

Note, whenever you see the focus-indicator around an element it has the psuedo-class 'focus' and 'focus-visible' applied. It's a bit confusing since for UX describe it as a single 'focus' state - though we need to apply some styles through the focus psuedo-class (and usually these are same as hover so we can declare these together). And then we need to apply the focus indicator styles using the focus-indicator psudo-class to override the browser. (Though for most components we do this via global styles).

Bit more here: https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible#focus_vs_focus-visible

![Screenshot 2024-05-13 at 11 49 00](https://github.com/bbc/simorgh/assets/98817636/0c26b6ed-9b3c-483a-8366-14cab8493ec2)